src/utils/components/CapacityInput/CapacityInput.tsx

```diff
@@ -91,10 +98,12 @@ const CapacityInput: FC<CapacityInputProps> = ({
         </SplitItem>
       </Split>
       <FormGroupHelperText validated={validated}>
-        {validated === ValidatedOptions.error &&
-          t('Size cannot be {{errorValue}}', {
-            errorValue: value < 0 ? 'negative' : 'zero',
-          })}
+        {validated === ValidatedOptions.error && (
+          <>
+            {t('Size cannot be {{errorValue}}', { errorValue: getErrorValue() })}
```

Comment:
No. I think you have to translate it like this
```suggestion
            {<>t('Size cannot be') {' '} {getErrorValue()}</>)}
```

and then 

```
    if (value > 0) {
      return t('less than');
    }
    return value < 0 ? t('negative') : t('zero');
```

To detect those translations.


Please if `yarn i18n` is not able to detect those translations, add 
```

      // t('less than')
    // t('negative')
    // t('zero')
    ```
    Somewhere so that it will pick it up and it will add them in the translation JSON file